consensus/src/dag/dag_state_sync.rs

```diff
@@ -108,16 +108,16 @@ impl StateSyncTrigger {
     async fn notify_commit_proof(&self, ledger_info: &LedgerInfoWithSignatures) {
         // if the anchor exists between ledger info round and highest ordered round
         // Note: ledger info round <= highest ordered round
-        if self
+        let local_highest_committed_round = self
             .ledger_info_provider
-            .get_highest_committed_anchor_round()
-            < ledger_info.commit_info().round()
-            && self
-                .dag_store
-                .read()
-                .highest_ordered_anchor_round()
-                .unwrap_or_default()
-                >= ledger_info.commit_info().round()
+            .get_highest_committed_anchor_round();
+        let local_highest_ordered_round = self
+            .dag_store
+            .read()
+            .highest_ordered_anchor_round()
+            .unwrap_or_default();
+        if local_highest_committed_round < ledger_info.commit_info().round()
+        // && local_highest_ordered_round < ledger_info.commit_info().round()
```

Comment:
probably leave a todo here to not send duplicate ones

Reply:
@sitalkedia told me deduping affected performance.

Reply:
added a todo